enacts/calc.py

```diff
@@ -231,6 +260,71 @@ def sql2geom(shapes_sql, db_config):
     return df
 
 
+def synthesize_geom(bbox, level=0):
```

Comment:
I suggest not defining a default value for `level`. Removing the default won't prevent you from calling it in keyword style (`synthesize_geom(config, level=level`).